### DISCUSSION TITLE
Troubleshooting Timed Out Builds

### DISCUSSION THREAD

**Discussion by stephencelis** [SwiftPackageIndex/SwiftPackageIndex-Server#2615] [Q&A]
Is there a way to figure out why builds are timing out? Maybe a way to get a partial log?

We have a project with visionOS and watchOS builds timing out:

https://swiftpackageindex.com/pointfreeco/swift-snapshot-testing/builds

Would love to track the issue down, as we are able to build for both platforms locally.

**Comment by finestructure** [comment]
Thanks for the report, @stephencelis ! I'll re-trigger these and run them locally with our setup to see what's going on. I believe typically this happens during package resolution and might be a machine-specific problem.

**Reply by stephencelis** [reply]
Thanks, Sven!
